.gitignore

```diff
@@ -20,6 +20,7 @@ pip-wheel-metadata
 .pytest_cache
 .tmp
 .ruff_cache
+.hypothesis
```

Comment:
Does this mean you are adding hypothesis to the suite of tools used for testing?

Reply:
No, but for some reason I keep getting a hypothesis directory when I run tests, so I wanted it to go away.